### DISCUSSION TITLE
Pacote para utilizar no lugar de lançamento de exception

### DISCUSSION THREAD

**Discussion by joao-sotero** [balta-io/forum#276] [Tira Dúvidas] [answered]
Ola, Alguem sabe em que aula o Balta comentou sobre o pacote que ele criou para utilizar no lugar do lançamento de exception para criar as mensagens de erros?

**Comment by joao-sotero** [comment]
@andrebaltieri

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @joao-sotero, como vai?
É este aqui:
https://balta.io/blog/flunt

No curso eu uso a versão 1.0.5:
```
dotnet add package Flunt --version 1.0.5
```

**Reply by joao-sotero** [reply]
Opa @andrebaltieri , exatamente isso que eu procurava. Consegue me lembrar em qual curso utilizou esse pacote?

**Reply by andrebaltieri** [reply]
Boa tarde, @joao-sotero, nos cursos Modelando Domínios Ricos e Refatorando para Testes de Unidade.
